src/styles/skeleton.scss

```diff
@@ -0,0 +1,76 @@
+@import './new-settings';
+@import './mixins';
+
+$block: fd-skeleton;
+
+.#{$block} {
+  @include fd-reset();
```

Comment:
why do you need the reset?
I see cases like this: `fd-button fd-button--primary fd-skeleton`, which means that `fd-skeleton` can be/is a "modifier" for another component, and by adding the reset here you are changing some of the properties of the main component, such as margins, paddings, etc. 
Is this desired behavior?